sdks/python/apache_beam/runners/worker/opcounters.py

```diff
@@ -202,6 +205,17 @@ def __init__(
     self._sample_counter = 0
     self._next_sample = 0
 
+    if consumers and len(consumers):
```

Comment:
You can just do `if consumers` here.

As discussed, there may be more than one consumer, so worth thinking about what to do in that case.